src/main/java/uk/gov/register/views/EntryListView.java

```diff
@@ -95,6 +95,12 @@ public String entriesTo(final String mediaType, final Provider<RegisterName> reg
         return recordKey;
     }
 
+    @SuppressWarnings("unused, used from templates")
+    public String getDownloadLink() {
+        Optional<String> recordKey = getRecordKey();
+        return recordKey.isPresent() ? String.format("record/%s/entries", recordKey.get()) : "entries"; 
```

Comment:
could you just call `getRecordKey()` directly here?

Reply:
I can but then that's two calls rather than one, which is why I didn't. Don't mind either way though.

Reply:
OK makes sense, happy to leave it as-is!